tensorflow/core/kernels/mkl/mkl_kernel_util.h

```diff
@@ -19,6 +19,7 @@ limitations under the License.
 #ifdef INTEL_MKL
 
 #include "dnnl.hpp"
+#include "tensorflow/core/framework/types.pb.h"
```

Comment:
Apparently we can't import protobuf headers in kernels.  Why is this needed?  Is there a potential work-around?

Reply:
Corrected.